### ISSUE TITLE
Failed: Setup and Call timeout >540.0s

### ISSUE THREAD

**Issue by palango** [raiden-network/raiden#4493]
```
________________ test_payment_events_endpoints[matrix-False-3] _________________

>       lambda: ihook(item=item, **kwds),
        when=when,
    )

../venv-3.7-LINUX/lib/python3.7/site-packages/flaky/flaky_pytest_plugin.py:306: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
raiden/tests/conftest.py:439: in pytest_runtest_call
    if should_print:
/usr/local/lib/python3.7/contextlib.py:119: in __exit__
    next(self.gen)
raiden/tests/conftest.py:312: in timeout_for_setup_and_call
    report()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def report():
        gevent.util.print_run_info()
>       pytest.fail(f"Setup and Call timeout >{item.timeout_setup_and_call}s")
E       Failed: Setup and Call timeout >540.0s

raiden/tests/conftest.py:272: Failed
``` 

Logs:
[raiden-debug_2019-07-30T14:13:39.701472.log](https://github.com/raiden-network/raiden/files/3447267/raiden-debug_2019-07-30T14.13.39.701472.log)

**Comment by LefterisJP** [comment]
I will take a look at the logs of this one. Can't reproduce locally.

**Comment by ulope** [comment]
Possibly fixed by #4495

**Comment by LefterisJP** [comment]
I checked the logs and I believe that #4495 may fix this. 

 ```
error    this=<MatrixTransport node:0xB7BE08AFBCB46bb27bb3404004C8ceB30B25161a id:140229923305040> running=1 subtask=<Greenlet "Greenlet-1" at 0x7f89d2f75290: <bound method UserAddressManager.get_address_mgr_info of <raiden.network.transport.matrix.utils.UserAddressManager object at 0x7f89d2c1b210>>> exc=KeyError('@0x26b0def70e4338b8ad131ea969e287c3864061e0:localhost:8511')
```

The above entry is in the logs and the matrix transport dies.
Right after that the test keeps on going with just mining blocks until it's killed by the timeout.

I would close this for now, since it's also not reproducible. Reopen if it's seen again.
